components/subchartsdemo/README.md

```diff
@@ -0,0 +1,255 @@
+# Example: subcharts
+
+This folder contains an example how to deploy an application consistsing of multiple helm charts. It consists of a helm chart, a [toi package](../../docs/reference/ocm_toi-bootstrapping.md) and a reference to the [helm helminstaller](../helminstaller/README.md).
+
+The helm chart has multiple other charts as dependencies and has no templates. Its only purpose is to act as an anchor for deployment. The toi package is used to install the component with the helm-installer using the `ocm bootstrap` command.
+
+## Helm Charts with dependencies
+Goal of this deployment is to install two different components having their own helm-charts: echoserver and pod-info. Both have their own component version and a helm-chart resource. The chart dependencies are declared in the `packagespec.yaml` as `subcharts`:
+
+```yaml
+executors:
+  - ...
+    config:
+      ...
+      subcharts:
+        podinfo:
+          resource:
+            name: podinfo-chart
+          referencePath:
+          - name: podinfo
+        echoserver:
+          resource:
+            name: echo-chart
+          referencePath:
+          - name: echoserver
+```
+For echoserver a dependency is declared to a resource named `echo-chart` contained in the reference named `echoserver` of this component version (`referencePath`):
+
+```yaml
+components:
+- name: .../subcharts
```

Comment:
Wouldn't it be more readable if we put concrete names here?

Reply:
component name ist now fully specified.